Hibernate/OneToOneMapping/Instructor.java

```diff
@@ -0,0 +1,92 @@
+package OneToOneMapping;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="instructor")
+public class Instructor {
+
+	@Id
+	@GeneratedValue(strategy=GenerationType.IDENTITY)
```

Comment:
Try using GenerationType.AUTO and tell me the difference

Reply:
Persistence provider will automatically pick an appropriate strategy if we use AUTO generation Type.As we created table with auto increment here the persistence provider assign primary keys using identity column mentioned.So we used IDENTITY here